tests/TestRunner.elm

```diff
@@ -3,11 +3,12 @@ port module TestRunner exposing (..)
 import Tests
 import Test.Runner.Node exposing (run, TestProgram)
 import Json.Encode exposing (Value)
+import Sample
```

Comment:
This is making the [tests fail](https://travis-ci.org/AndelaOSP/shufflebox-frontend/jobs/219171442). Maybe you committed it by mistake? 🙂